test/test_flake8.py

```diff
@@ -23,7 +23,7 @@ def test_flake8_conformance():
     argv = [
         '--ignore=%s' % ','.join([
             'D100', 'D101', 'D102', 'D103', 'D104', 'D105',
-            'E501']),
+            'E501', 'W504']),
```

Comment:
Use `--extend-ignore` instead.

Reply:
This is only available in flake8 3.7, but since I updated to use that, it should work.